CMakeLists.txt

```diff
@@ -200,11 +202,10 @@ if(WIN32 AND NOT CYGWIN)
   set(default_header_install_dir include/)
   set(default_auxiliary_install_dir share/)
 else()
-  string(TOLOWER ${PROJECT_NAME} PROJECT_NAME_LOWER)
   set(default_runtime_install_dir bin/)
-  set(default_library_install_dir lib/${PROJECT_NAME_LOWER})
-  set(default_archive_install_dir lib/${PROJECT_NAME_LOWER})
-  set(default_header_install_dir include)
+  set(default_library_install_dir lib/)
```

Comment:
Thanks, I still had  the old files lying around when running the tests.

On August 11, 2016 2:05:33 PM GMT+02:00, Jeff DiClemente <notifications@github.com> wrote:
>>    set(default_runtime_install_dir bin/)
>> -  set(default_library_install_dir lib/${PROJECT_NAME_LOWER})
>> -  set(default_archive_install_dir lib/${PROJECT_NAME_LOWER})
>> -  set(default_header_install_dir include)
>> +  set(default_library_install_dir lib/)
>
>This change breaks the ```framework\examples\makefile``` example.
>```framework\examples\makefile\Makefile``` needs to be changed as well.
>
>-- 
>You are receiving this because you were assigned.
>Reply to this email directly or view it on GitHub:
>https://github.com/CppMicroServices/CppMicroServices/pull/122/files/a818fb88a0cbb6a80d255ab73e7ce37716c491aa..c1e7629b0a8b4fc4a235920c08e2a59cfe874f73#r74409731

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.